src/modules/createChannel/CreateChannel.tsx

```diff
@@ -1,128 +1,340 @@
 // This is the Parent component for the Create Channel flow.
+// React + web3 essentials
+import { useEffect, useState } from 'react';
+
+// Third party libraries
+import { useFormik } from 'formik';
+import { ethers } from 'ethers';
+
+// Hooks
```

Comment:
Also fix the channel creation. it keeps on loading. User should be able to see the channel dashboard after the channel is created.

Reply:
We can take this into action once the new channel dashboard is up